experimental/dds/tree/src/id-compressor/IdCompressor.ts

```diff
@@ -839,9 +842,9 @@ export class IdCompressor {
 
 		if (isLocal) {
 			this.logger?.sendTelemetryEvent({
-				eventName: 'IdCompressor:IdCompressorStatus',
-				eagerFinalIdCount,
-				localIdCount: remainingCount,
+				eventName: 'SharedTreeIdCompressor:IdCompressorStatus',
```

Comment:
will there be an identical PR into the other compressor?

Reply:
Yes, these changes will need to be ported to the other compressor as well. We'll use a different namespace in that one (and the tests will need to be ported to not use chai) to distinguish the data. Could tack it on to this but I thought it would be confusing.

Reply:
I'm indifferent. Either way.